controllers/C_Prescription.class.php

```diff
@@ -651,253 +651,253 @@ function multiprintcss_action($id = "") {
+                $phar = new Pharmacy($_POST['pharmacy_id']);
+                //print_r($phar);
+                if ($phar->get_transmit_method() == TRANSMIT_PRINT) {
+                    return $this->_print_prescription($p, $dummy);
+                }
+                elseif ($phar->get_transmit_method() == TRANSMIT_EMAIL) {
+                    $email = $phar->get_email();
+                    if (!empty($email)) {
+                        return $this->_email_prescription($p,$phar->get_email());
+                    }
+                    //else print it
+                }
+                elseif ($phar->get_transmit_method() == TRANSMIT_FAX) {
+                    $faxNum= $phar->get_fax();
+                    if(!empty($faxNum)) {
+                        Return $this->_fax_prescription ($p,$faxNum);
+                    }
+                    // return $this->assign("process_result","No fax server is currently setup.");
+                    // else default is printing,
+                }
+                else {
+                    //the pharmacy has no default or default is print
+                    return $this->_print_prescription($p, $dummy);
+                }
+                break;
+        }
+
+        return;
+
+    }
+
+    function _print_prescription($p, & $toFile) {
+        require_once ($GLOBALS['fileroot'] . "/library/classes/class.ezpdf.php");
+        $pdf = new Cezpdf($GLOBALS['rx_paper_size']);
+        $pdf->ezSetMargins($GLOBALS['rx_top_margin']
+            ,$GLOBALS['rx_bottom_margin']
+            ,$GLOBALS['rx_left_margin']
+            ,$GLOBALS['rx_right_margin']
+        );
+
+        $pdf->selectFont($GLOBALS['fileroot'] . "/library/fonts/Helvetica.afm");
+
+        // Signature images are to be used only when faxing.
+        if(!empty($toFile)) $this->is_faxing = true;
+
+        $this->multiprint_header($pdf, $p);
+        $this->multiprint_body($pdf, $p);
+        $this->multiprint_footer($pdf);
+
+        if(!empty($toFile)) {
+            $toFile = $pdf->ezOutput();
+        }
+        else {
+            $pdf->ezStream();
+            // $pdf->ezStream(array('compress' => 0)); // for testing with uncompressed output
+        }
+        return;
+    }
 
         function _print_prescription_css($p, & $toFile) {
 
                 $this->multiprintcss_preheader();
                 $this->multiprintcss_header($p);
                 $this->multiprintcss_body($p);
                 $this->multiprintcss_footer();
-	        $this->multiprintcss_postfooter();
+            $this->multiprintcss_postfooter();
 
         }
 
-	function _print_prescription_old($p, & $toFile) {
-		require_once ($GLOBALS['fileroot'] . "/library/classes/class.ezpdf.php");
-		$pdf = new Cezpdf($GLOBALS['rx_paper_size']);
-		$pdf->ezSetMargins($GLOBALS['rx_top_margin']
+    function _print_prescription_old($p, & $toFile) {
+        require_once ($GLOBALS['fileroot'] . "/library/classes/class.ezpdf.php");
```

Comment:
This commit will stop on the others that I have had rotting for a while.  I will have to start over, or dump them if we commit this.